mappings/net/minecraft/loot/condition/LootCondition.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_5341
+CLASS net/minecraft/class_5341 net/minecraft/loot/condition/LootCondition
```

Comment:
Was this not matched? :concern:

Reply:
Some mapped methods were moved in this class

Reply:
Yes, but loot conditions were an existing thing already 🤔

Reply:
Yes, but it's a new interface used by loot condition types, named after interface used by loot functions

Reply:
Ah, turned it into an interface? Interesting.

Reply:
In hindsight it might have been worth matching, however it would have broken anyway.